SSA/Projects/DCE/DCE.lean

```diff
@@ -225,7 +224,7 @@ def arglistDeleteVar? {Γ: Ctxt d.Ty} {delv : Γ.Var α} {Γ' : Ctxt d.Ty} {ts :
         ⟩
 
 /- Try to delete a variable from an Expr -/
-def Expr.deleteVar? (DEL : Deleted Γ delv Γ') (e: Expr d Γ .pure t) :
+def Expr.deleteVar? [DialectSignature d] [DialectDenote d] [Monad d.m] (DEL : Deleted Γ delv Γ') (e: Expr d Γ .pure t) :
```

Comment:
Are all these changes really neccessary? I think they make the code more verbose. I added back the instances to the variables declaration and removed them from this definition, and there was no error when I ran it locally.

As far as I understand it, definitions were not affected by the change to "variables", but perhaps I am mistaken on this.